### PR TITLE
Add Heat collection function

### PR DESCRIPTION
**Pull request by bshephar**

### CODE REVIEW

File: collection-scripts/gather_services_status [hunk 1]
```diff
@@ -74,6 +77,14 @@ get_cinder_status() {
     ${BASH_ALIASES[os]} volume backend pool list --long > "$CINDER_PATH"/pool_list
 }
 
+# Heat service gathering - services and stacks
+get_heat_status() {
+    local HEAT_PATH="$BASE_COLLECTION_PATH/ctlplane/heat"
+    mkdir -p "$HEAT_PATH"
+    ${BASH_ALIASES[os]} orchestration service list > "$HEAT_PATH"/service_list
+    ${BASH_ALIASES[os]} stack list > "$HEAT_PATH"/stack_list
```
Comment (rabi) [inline collection-scripts/gather_services_status:85]: What user and project are we going to use for this command? Though you can use ` --all-projects` with admin user to list all stacks, I'll probably drop this.

Comment (bshephar) [reply collection-scripts/gather_services_status:85]: Yeah, we can always just ask them for stack specific information if that's what's required. I think the main thing is that we have the logs from the pods, and the config from the config maps. Let's drop this.

**Review by lewisdenny — APPROVED** [review]
Looks like a good addition to me +1

**Review by fmount — APPROVED** [review]

**Review by gibizer — APPROVED** [review]
Looks good

**Pull request merged**